src/translations/de.json

```
@@ -0,0 +1,17 @@
+{
+	"Sort Ascending": "Aufsteigend sortieren",
+	"Sort Descending": "Absteigend sortieren",
+	"Reset sorting": "Sortierung zurücksetzen",
+	"Remove column": "Spalte entfernen",
+	"No Data": "Keine Daten",
+	"{count} cells copied": {
+		"0": "{count} Zellen kopiert",
+		"1": "{count} Zelle kopiert",
+		"default": "{count} Zellen kopiert"
+	},
+	"{count} rows selected": {
+		"0": "{count} Zeilen ausgewählt",
+		"1": "{count} Zeile ausgewählt",
+		"default": "{count} Zeilen ausgewählt"
```

```suggestion
		"1": "{count} Zelle kopiert",
		"default": "{count} Zellen kopiert"
	},
	"{count} rows selected": {
		"1": "{count} Zeile ausgewählt",
		"default": "{count} Zeilen ausgewählt"
```

Translation for 0 is the same as default.